client/reader/recent/style.scss

```diff
@@ -55,108 +57,90 @@
-			// if no media is given.
-			// The presence of this element adds a gap on the left side of the element,
-			// throwing off the padding.
-			.dataviews-view-list__media-wrapper {
-				display: none;
-			}
-
-			.dataviews-view-list__item-wrapper {
-				justify-content: start;
-
-				[role="gridcell"] {
-					width: 100%;
+			li {
+				border-top: 1px solid var( --studio-gray-0 );
+				
+				&.is-selected + li {
+					border-top: 1px solid var( --studio-gray-0 );
 				}
-			}
-
-			.dataviews-view-list__field-wrapper {
-				width: 100%;
-			}
-
-			.dataviews-view-list__fields {
-				display: flex;
-				align-items: center;
-				gap: $recent-feed-spacing;
-				padding: 0 $recent-feed-spacing;
-			}
 
-			.dataviews-view-list__field:has(.recent-post-field) {
-				overflow: hidden;
-				flex-grow: 1;
-			}
-		}
-
-		.recent-seen-field {
-			padding: $recent-feed-spacing 0;
+				&:hover {
+					cursor: pointer;
+					background-color: #f7faff;
+				}
 
-			img {
-				border-radius: 50%;
-			}
+				&.is-selected {
+					border-top: 1px solid #3858e940;
 
-			svg:hover {
-				fill: unset;
-			}
-		}
+					.dataviews-view-list__item-wrapper {
+						background-color: #f7faff;
+					}
+				}
 
-		.recent-post-field {
-			display: flex;
-			justify-content: space-between;
-			align-items: center;
-			text-align: left;
-			padding: $recent-feed-spacing 0;
-			width: 100%;
-			min-height: fit-content;
-			gap: $recent-feed-spacing;
-
-			%text-shared {
-				white-space: nowrap;
-				text-overflow: ellipsis;
-				overflow: hidden;
-			}
+				.components-flex {
+					align-items: center;
+				}
 
-			&__title {
-				min-width: 0;
-			}
+				.dataviews-view-list__media-wrapper {
+					border-radius: 50%;
+					width: 24px;
+					height: 24px;
+					background-color: transparent;
 
-			&__title-text {
-				font-weight: 700;
-				font-size: rem(13px);
-				@extend %text-shared;
-			}
+					&::after {
+						box-shadow: none;
+					}
+				}
 
-			&__site-name {
-				font-weight: 400;
-				font-size: rem( 11px );
-				@extend %text-shared;
+				.recent-post-field {
+					display: flex;
+					justify-content: space-between;
+					align-items: center;
+					text-align: left;
+					width: 100%;
+					min-height: fit-content;
+					gap: $recent-feed-spacing;
+		
+					%text-shared {
+						white-space: nowrap;
+						text-overflow: ellipsis;
+						overflow: hidden;
+					}
+		
+					&__title {
+						min-width: 0;
+					}
+		
+					&__title-text {
+						font-weight: 700;
+						font-size: rem(13px);
+						@extend %text-shared;
+					}
+		
+					&__site-name {
+						font-weight: 400;
+						font-size: rem( 11px );
+						line-height: 1;
+						@extend %text-shared;
+					}
+		
+					&__featured-image {
+						max-width: 38px;
+						align-self: flex-end;
+						flex-shrink: 0;
+					}
+				}
 			}
 
-			&__featured-image {
-				max-width: 38px;
-				align-self: flex-end;
-				flex-shrink: 0;
+			.dataviews-view-list__item {
```

Comment:
This style changes the padding of the DataViews items to match the design / maximize horizontal space.